server/job/job.js

```diff
@@ -34,6 +34,7 @@ class Job {
     this.processed = 0
     this.result = {}
     this.errors = {}
+    this.context = {} // object shared among nested jobs
```

Comment:
job starts to have  a lot of properties.. should user, surveyId, survey, and/or others be props of context?